Formula/smartlook-crash-reports-cli.rb

```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class SmartlookCrashReportsCli < Formula
+  desc "CLI tool for uploading crash mapping files to the Smartlook"
+  homepage "https://www.npmjs.com/package/@smartlook/smartlook-crash-reports-cli"
+  url "https://registry.npmjs.org/@smartlook/smartlook-crash-reports-cli/-/smartlook-crash-reports-cli-2.1.3.tgz"
+  sha256 "77e4f67721be3cd526e49a38967ebd35549386a22cae06be1d2d89f1e65c0f5a"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    output = shell_output("#{bin}/smartlook-crash-reports --version")
+    assert_match("2.1.3", output.strip)
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.